### PR TITLE
Add name or uuid to footer

### LINKED ISSUE #813: Add server's identity to admin page
I am dealing with remote access to more servers than just mine - the fact that the admin page of each server looks exactly the same and their browser tab titles offer no clue is not helping.

- add vessel's name to html title
- add vessel's name & id to header or footer or both

### PR DESCRIPTION
**Pull request by wouterdebie**
This fixes #813

### CODE REVIEW

File: packages/server-admin-ui/src/components/Footer/Footer.js [hunk 1]
```diff
@@ -2,38 +2,57 @@ import React, { Component } from 'react'
 import { connect } from 'react-redux'
 import { Link } from 'react-router-dom'
 
+function fetchVessel() {
+  fetch(`/vessel`, {
+    credentials: 'include'
+  })
+    .then(response => response.json())
+    .then(data => {
+      this.setState({ ...data, hasData: true })
```
Comment (tkurki) [inline packages/server-admin-ui/src/components/Footer/Footer.js]: hasData is not used anywhere?

File: packages/server-admin-ui/src/components/Footer/Footer.js [hunk 2]
```diff
@@ -2,38 +2,57 @@ import React, { Component } from 'react'
...
     }
+    this.fetchVessel = fetchVessel.bind(this)
+  }
+
+  componentDidMount() {
+    this.fetchVessel()
   }
 
   render() {
+    console.log(this)
```
Comment (tkurki) [inline packages/server-admin-ui/src/components/Footer/Footer.js]: Please remove console.log

File: packages/server-admin-ui/src/components/Footer/Footer.js [hunk 3]
```diff
@@ -2,38 +2,57 @@ import React, { Component } from 'react'
...
-      )}
+          </span>
+        )}
         {this.props.loginStatus.status === 'loggedIn' && (
           <span className='ml-auto'>
             Logged in as {this.props.loginStatus.username}
           </span>
         )}
+
+        &nbsp;- {this.state.name || this.state.uuid}
```
Comment (tkurki) [inline packages/server-admin-ui/src/components/Footer/Footer.js]: MMSI is better than uuid (shorter, more permanent, better identifier), so insert it into the chain before uuid.

File: packages/server-admin-ui/src/components/Footer/Footer.js (hunk 1 above)
Comment (wouterdebie) [reply packages/server-admin-ui/src/components/Footer/Footer.js]: Totally true! Thanks

File: packages/server-admin-ui/src/components/Footer/Footer.js (hunk 2 above)
Comment (wouterdebie) [reply packages/server-admin-ui/src/components/Footer/Footer.js]: Oops, forgot this one. Thanks!

File: packages/server-admin-ui/src/components/Footer/Footer.js (hunk 3 above)
Comment (wouterdebie) [reply packages/server-admin-ui/src/components/Footer/Footer.js]: Allright, updated.

**Pull request merged**